README.md

```diff
@@ -3,6 +3,80 @@
 This repository contains sources of officially maintained Arnold applications,
 _a.k.a._ trays.
 
+## Requirements
+
+The only required dependency to work on this project is the `arnold` CLI. Check
+out [openfun/arnold](https://github.com/openfun/arnold) to install it.
```

Comment:
Oh, I didn't find the installation instructions. And then I found them in a [pull request](https://github.com/openfun/arnold/pull/582) :)

Reply:
Yeah, I should merge this PR and pursue my work later... I was too ambitious when willing to revamp the whole documentation. :grimacing: